src/App.tsx

```diff
@@ -1,18 +1,26 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Pagination } from './components/Pagination';
 import './App.css';
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [selectedValue, setSelectedValue] = useState(5);
+  const [active, setActive] = useState(1);
+  const optionValue = [3, 5, 10, 20];
+  const sliceItems = items.slice(
+    selectedValue * active - selectedValue, selectedValue * active,
+  );
+
   return (
     <div className="container">
       <h1>Items with Pagination</h1>
 
       <p className="lead" data-cy="info">
-        Page 1 (items 1 - 5 of 42)
+        {`Page ${active} `}
+        {`(${sliceItems[0].replace('m', 'ms').replace('I', 'i')} - ${sliceItems[sliceItems.length - 1].split(' ')[1]} of ${items.length})`}
```

Comment:
It would be great to create a separate variable for this value as now it looks a bit complicated